common-util/functions/errors.js

```diff
@@ -0,0 +1,46 @@
+import isObject from 'lodash/isObject';
+import { notifyError as notifyErrorFn } from '@autonolas/frontend-library';
+// create a function to specific error message based on error code
+export const notifyError = notifyErrorFn;
```

Comment:
why do we need this? I see one place where it's imported from here but in others it's still from '@autonolas/frontend-library'